include/roaring/roaring64.h

```diff
@@ -380,6 +381,92 @@ void roaring64_bitmap_andnot_inplace(roaring64_bitmap_t *r1,
 bool roaring64_bitmap_iterate(const roaring64_bitmap_t *r,
                               roaring_iterator64 iterator, void *ptr);
 
+/**
+ * Create an iterator object that can be used to iterate through the values.
+ * Caller is responsible for calling `roaring64_free_iterator()`.
+ *
+ * The iterator is initialized. If there is a value, then this iterator points
+ * to the first value and `roaring64_iterator_has_value()` returns true. The
+ * value can be retrieved with `roaring64_iterator_value()`.
+ */
+roaring64_iterator_t *roaring64_create_iterator(const roaring64_bitmap_t *r);
```

Comment:
I generally prefer (and CRoaring seems to _mostly_ follow) to name functions as `{prefix}_{noun}_{verb}`, so all the things you can do with say, an iterator, all share a common prefix.

I'd rather these functions be `roaring64_iterator_create`, `roaring64_iterator_advance`, etc.

However, I do see these names are consistent with the `roaring_uint32_iterator` functions.

@lemire, is that something we could change in a major version too, fix the names of those functions (and/or make the new functions, but make deprecated aliases for the original names)?

Reply:
> is that something we could change in a major version too, fix the names of those functions (and/or make the new functions, but make deprecated aliases for the original names)?

Sure.

Reply:
Renamed the roaring64 functions in this PR. I can add aliases for the 32-bit iterator functions in a separate PR.